sound/soc/intel/boards/sof_maxim_common.c

```diff
@@ -47,11 +47,11 @@ static int max98373_hw_params(struct snd_pcm_substream *substream,
 	for_each_rtd_codec_dais(rtd, j, codec_dai) {
 		if (!strcmp(codec_dai->component->name, MAX_98373_DEV0_NAME)) {
 			/* DEV0 tdm slot configuration */
-			snd_soc_dai_set_tdm_slot(codec_dai, 0x30, 3, 8, 16);
+			snd_soc_dai_set_tdm_slot(codec_dai, 0x3, 3, 8, 32);
```

Comment:
unify code style to 0x3, 0x3, 8, 32 ?  tx_mask and rx_mask should be in the same style.

Reply:
sorry, I just pushed wrong local change recently, this is for debug only so should not go to here :)

Reply:
removed. @RanderWang